packages/core/src/index.ts

```diff
@@ -22,13 +22,14 @@ interface Config {
 // The journey manager object
 export interface VoiceCompass {
   updateStep: (data: StepData) => Promise<StepUpdate>;
-  getLastStepId: () => string | null;
+  getLastStepId: () => string | null | undefined;
```

Comment:
What is the purpose of `null` and `undefined`, something to do with intentionality? I would just stick to `null` always.

Reply:
this change is related to making `stepId` optional. I can remove `null` and leave `undefined`

Reply:
`lastStepId = stepData.stepId || null` so it's a single type of value would be easier, wouldn't it?